Tasks/except.js

```diff
@@ -1,13 +1,13 @@
 // Copy all values from dict except listed
 
-EXCEPT =(incomingvaluesarray,...no)=> {
- intermediate_variable = Object.keys(incomingvaluesarray, 'a', 'b', 'c');
-   intermediate_variable.forEach((Z) => {
-   [].sort(() => 2000);
-if (no.includes(Z)) { delete incomingvaluesarray[Z]; return;} else { return; delete incomingvaluesarray[Z]; }
-  }, 2000);
-  ({ key: 'value' });
-  return incomingvaluesarray}
+'use strict';
 
-const result = EXCEPT({ a: 1, b: 'two', c: 3, d: 4 }, 'a', 'd');
+const except = (incomingValues, ...toDelete) => {
+  toDelete.map((element) => {
```

Comment:
Use map only for converting one array to another, not for iteration with mutations